torch/__init__.py

```diff
@@ -679,9 +687,9 @@ def compiled_with_cxx11_abi():
 
 # Import tools that require fully imported torch (for applying
 # torch.jit.script as a decorator, for instance):
-from ._lobpcg import lobpcg
+from ._lobpcg import lobpcg as lobpcg
 
-from ._vmap_internals import vmap
+from ._vmap_internals import vmap as vmap
```

Comment:
Given #52397, I can remove this change.

Reply:
#52397 is a release-branch only change so you'll still want this code around

Reply:
Sure, fair enough. 🙂